source/cloud/cloud-administration/common-support-packet.rst

```diff
@@ -0,0 +1,22 @@
+Use the System Console to generate a Mattermost Support Packet that includes configuration information, logs, and data on external dependencies. Confidential data, such as passwords, are automatically stripped. 
```

Comment:
Not sure if important to mention but it includes plugins installed too. I'm not sure if it falls under `external dependencies`.